lighthouse-cli/test/smokehouse/test-definitions/a11y/expectations.js

```diff
@@ -134,7 +134,7 @@ const expectations = [
                   },
                   'selector': 'body > section > div > div#aria-treeitem-name',
                   'snippet': '<div id="aria-treeitem-name" role="treeitem">',
-                  'explanation': 'Fix any of the following:\n  Element does not have text that is visible to screen readers\n  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty\n  aria-label attribute does not exist or is empty\n  Element has no title attribute',
+                  'explanation': 'Fix any of the following:\n  Element does not have text that is visible to screen readers\n  aria-label attribute does not exist or is empty\n  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty\n  Element has no title attribute',
```

Comment:
these are just rearranging of the `explanation` concat order. We could drop them from the expectations...but they were also really easy to c/p. I just did that for now.